test/test_main.cpp

```diff
@@ -299,6 +299,11 @@ void RelatedGL(const Manifold& out, const std::vector<MeshGL>& originals,
   ASSERT_FALSE(out.IsEmpty());
   const ivec3 normalIdx = updateNormals ? ivec3(3, 4, 5) : ivec3(0);
   MeshGL output = out.GetMeshGL(normalIdx);
+
+  float epsilon = std::max(static_cast<float>(out.GetEpsilon()),
+                           std::numeric_limits<float>::epsilon() *
+                               static_cast<float>(out.BoundingBox().Scale()));
```

Comment:
Why is this necessary?

Reply:
This is getting MeshGL, which uses single precision. The epsilon value we track is for double precision, which is too small here.

Reply:
Hmm, that sounds like a bug we should fix at the MeshGL level. Also, I thought I already had?